planner/core/exhaust_physical_plans.go

```diff
@@ -1718,7 +1722,7 @@ func (la *LogicalAggregation) getHashAggs(prop *property.PhysicalProperty) []Phy
 		return nil
 	}
 	hashAggs := make([]PhysicalPlan, 0, len(wholeTaskTypes))
-	taskTypes := []property.TaskType{property.CopSingleReadTaskType, property.CopDoubleReadTaskType, property.CopTiflashTaskType}
+	taskTypes := []property.TaskType{property.CopSingleReadTaskType, property.CopDoubleReadTaskType, property.CopTiFlashLocalReadTaskType, property.CopTiFlashGlobalReadTaskType}
```

Comment:
Why hash agg require TiFlash Global Read ?

Reply:
I just extends the origin logical, and I think this will be refined when supporting multi join pushdown